.changelog/9198.txt

```diff
@@ -0,0 +1,33 @@
+```release-note:improvement
+cli: add Usage: nomad [-version] [-help] [-autocomplete-(un)install] <command> [args]
```

Comment:
are we expecting the entire Nomad usage here?

Reply:
LOL, I did this at my command line:

```
 new-cl improvement 9198 "cli: add `nomad config validate` command to check configuration files without an agent"
```

And then didn't look at the resulting file. 😊